components/dashboard/src/login/SSOLoginForm.tsx

```diff
@@ -19,7 +19,12 @@ type Props = {
 };
 
 function getOrgSlugFromPath(path: string) {
-    return path.split("/")[2];
+    // '/login/acme' => ['', 'login', 'acme']
+    const pathSegments = path.split("/");
+    if (pathSegments[1] !== "login") {
+        return;
+    }
+    return pathSegments[2];
```

Comment:
Might be easier to follow by doing
* trim leading slash
* Split
then indices are slightly less confusing